### ISSUE TITLE
Cypress desktop GUI delivers different test results than the Cypress CLI test env.   

### ISSUE THREAD

**Issue by sully1313** [cypress-io/cypress#1252]
- Operating System: OSX
- Cypress Version:1.4.1
- Browser Version: Chrome

### Is this a Feature or Bug?
Assuming it is a bug

### Current behavior:
In these tests I am testing a react search-dropdown component.
![screen shot 2018-02-01 at 7 50 53 pm](https://user-images.githubusercontent.com/16328122/35754241-996c2f44-0817-11e8-986b-4aaffc017275.png)

All tests are currently passing in the cypress GUI test env as seen above. However, I cannot say the same for the cypress terminal test environment. All the same tests are currently failing in this env. 
![screen shot 2018-02-01 at 7 50 08 pm](https://user-images.githubusercontent.com/16328122/35754265-b7a24f2a-0817-11e8-8c01-2f313dfa07bf.png)

I believe that in the terminal test env cypress is not able to trigger the dropdowns on the autocomplete component. Any suggestions?

### Desired behavior:
The desired behavior is that both the Cypress Desktop GUI & CLI issue the same test results

### How to reproduce:
Run the below code on an react autocomplete component

#### Test code:

```js
it('should select Residential button', () => {
    cy.get('button').within(() => {
      cy.get('div').contains('Residential').click()
    })
  })
  it('should enter a Residential address', () => {
    cy.get('input').type('525 Middlefied Rd Menlo Park')
    cy.get('div').contains('525 Middlefield Road, Menlo Park, CA, United States').click()
  })
```

**Comment by brian-mann** [comment]
When running headlessly we are running Electron which is based on Chromium 53 - which is different than your local browser and this can cause differences.

Read more here: https://docs.cypress.io/guides/guides/launching-browsers.html#

I suggest running your tests in Electron (in headed mode) so that they are the same - or just simply run your tests with the `--browser chrome` argument to get the same behavior.

**Comment by sully1313** [comment]
@brian-mann Thank you! This solved our issue.

**Comment by brian-mann** [comment]
Released in [`2.0.0`](https://docs.cypress.io/guides/references/changelog.html#2-0-0).
